### DISCUSSION TITLE
Use tmpfs for /tmp to speed up Recognize

### DISCUSSION THREAD

**Discussion by rekcodocker** [nextcloud/all-in-one#3094] [Questions] [answered]
### Is your feature request related to a problem? Please describe. 
The 'recognize' app suggests to use tmpfs for /tmp as it reduces disk load and speeds up the recognize module.

link: [https://github.com/nextcloud/recognize](https://github.com/nextcloud/recognize)

This is what they propose for Docker Compose

```
  app:
    image: nextcloud:26
    ...
    volumes:
      - type: tmpfs
        target: /tmp:exec
      ...
    ...
```

or this for ordinary Docker:
docker run: `Add --mount type=tmpfs,destination=/tmp:exec` to command line.

I use docker compose to start Nextcloud All In One.
It starts the mastercontainer, which then downloads, configures and maintains the other containers.
I do not want to interfere in this process.
I think this modification applies to the Nextcloud-container.  I can not judge if things would break if /tmp was using tmpfs.

This would bring 'recognize' and AIO closer together.

**Comment by Zoey2936** [comment, marked as answer]
already done: https://github.com/nextcloud/all-in-one/pull/2981
